### DISCUSSION TITLE
Meshtastic

### DISCUSSION THREAD

**Discussion by qwertyuiop123234567uhgfv** [community/community#132461] [Copilot Conversations]
### Select Topic Area

Question

### Body

When I build the firmware for Meshtastic using the platformio plugin in VScode, I get this error, the file in this error exists in the folder, but it still reports that the file can't be found, how do I fix it?
<img width="829" alt="屏幕截图 2024-07-16 114358" src="https://github.com/user-attachments/assets/af9606f7-4fda-499a-945a-c8fc5e9b0155">

**Comment by Akash1134** [comment]
Hi @qwertyuiop123234567uhgfv ,

Thanks for being a part of the GitHub Community, we're glad you're here!

We'd recommend reporting this issue in the [vscode repo](https://github.com/microsoft/vscode/issues).

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
